unit_tests/test_events.py

```diff
@@ -338,3 +340,31 @@ def test_failed_stall_during_filter(self):
                                line="[99.80.124.204] [stdout] Mar 31 09:08:10 warning|  reactor stall 20").publish()
 
         self.assertEqual(event.severity, Severity.DEBUG)
+
+    def test_duration_format(self):
+        event = NodetoolEvent(nodetool_command="scrub --skip-corrupted drop_table_during_repair_ks_0", node='1.0.0.121',
+                              options="more options", publish_event=False)
+
```

Comment:
Need also to cover case when duration is not provided, which is supported by the function.

Reply:
added